schema.sql

```diff
@@ -1,8 +1,9 @@
 CREATE TABLE animals (
-    id int primary key not null,
+    id SERIAL PRIMARY KEY not null,
     name varchar(100) not null,
     date_of_birth date not null,
     escape_attempts int not null,
     neutered bool not null,
-    weight_kg decimal not null
+    weight_kg decimal not null,
+    species varchar not null
```

Comment:
It would have been more organized and make your code cleaner,  if you had altered the the table by adding: 
```
      ALTER TABLE animals ADD species varchar(255);
```

Reply:
👍🏼